### PR TITLE
docs fix for doubleMean description

### PR DESCRIPTION
**Pull request by vtlim**
Fixes description for `doubleMean` aggregator.

This PR has:
- [x] been self-reviewed.

@sthetland @techdocsmith

### CODE REVIEW

**Review by suneet-s — APPROVED** [review]
Thanks for the contribution @vtlim! 

What do you think about pointing people to the quantiles aggregator for users who are looking for mean aggregation on ingestion https://druid.apache.org/docs/latest/development/extensions-core/datasketches-quantiles.html#aggregator

**Review by sthetland — APPROVED** [review]

**Comment by vtlim** [comment]
Thanks for the review, @suneet-s! I updated the description to include the reference to quantiles aggregator as you suggested.

I think we could use a better explanation (and/or an example) of the DataSketches aggregators in the docs; I'll look into this.
@techdocsmith

File: docs/querying/aggregations.md [hunk 1]
```diff
@@ -126,7 +126,9 @@ Computes and stores the sum of values as 32-bit floating point value. Similar to
 
 ### `doubleMean` aggregator
 
-Computes and returns arithmetic mean of a column values as 64 bit float value. This is a query time aggregator only and should not be used during indexing.
+Computes and returns arithmetic mean of a column values as 64 bit float value. `doubleMean` is a query time aggregator only. It is not available for indexing.
+
+To accomplish mean aggregation on ingestion, refer to the [Quantiles aggregator](https://druid.apache.org/docs/latest/development/extensions-core/datasketches-quantiles.html#aggregator) from the DataSketches extension.
```
Comment (sthetland) [inline docs/querying/aggregations.md:131]: The additional sentence looks good. Just to make this link more robust, we should use the following format:
  
```suggestion
To accomplish mean aggregation on ingestion, refer to the [Quantiles aggregator](../development/extensions-core/datasketches-quantiles.md#aggregator) from the DataSketches extension.
```

**Review by sthetland — CHANGES_REQUESTED** [review]

**Review by suneet-s — APPROVED** [review]
thanks for the contribution @vtlim

**Review by sthetland — APPROVED** [review]

**Pull request merged**